scripts/Capacitors_THT/make_Capacitors_THT.py

```diff
@@ -181,7 +181,7 @@
         makeResistorRadial(seriesname=seriesname, rm=rm, w=w, h=d, ddrill=ddrill, R_POW=R_POW,
                                     type=type, w2=w2, x_3d=[0, 0, 0], s_3d=[1, 1, 1], has3d=1,
                                     specialfpname="", add_description=add_description, name_additions=name_additions,
-                                    specialtags=specialtags, classname="C", lib_name="${KISYS3DMOD}/Capacitor_THT_Rectangular", script3d=scr, height3d=h3d)
+                                    specialtags=specialtags, classname="C", lib_name="Capacitor_THT", script3d=scr, height3d=h3d)

```

Comment:
Avoid prepending "${KISYS3DMOD}" in the library name as it is not part of it. The new generator script will take care of adding this to the 3D model path.

Change library name parameter to avoid library splitting.